### DISCUSSION TITLE
2023/02/10/rocksdb-asyncio-benchmark

### DISCUSSION THREAD

**Discussion by giscus** [wanghenshui/wanghenshui.github.io#92] [Q&A]
# 2023/02/10/rocksdb-asyncio-benchmark

https://wanghenshui.github.io/2023/02/10/rocksdb-asyncio-benchmark.html

**Comment by duanfuxiang0** [comment]
看官网的blog multi_get 和 scan 提升非常大

**Reply by wanghenshui** [reply]
我测的结果multii_get基本没发挥出来，5%不到。scan没有测试
